Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -38,4 +46,46 @@ public class ItemUseHandler : PacketHandler<GameSession> {
         
         session.Send(StoryBookPacket.Load(storyBookId));
     }
+
+    private static void HandleChatSticker(GameSession session, Item item) {
+        Dictionary<string, string> parameters = XmlParseUtil.GetParameters(item.Metadata?.Function?.Parameters);
+
+        if (!parameters.ContainsKey("id")) {
```

Comment:
I would also do the "id" parsing here to avoid crashes:
```cs
if (!parameters.ContainsKey("id") || !int.TryParse(parameters["id"], out int stickerSetId)) {
```